docs/contextual-bandits/actions_contexts.md

```diff
@@ -55,11 +55,11 @@ Then there might be specific requirements such as not showing any maternity item
 
 #### New actions
 
-Our Contextual Bandits automatically explore new actions as they are encountered, and over time will learn for which contexts these are most effective.
+Our Contextual Bandits automatically explores new actions as they are encountered and over time will learn for which contexts these actions are most effective.
```

Comment:
It should either be
`Our contextual bandit automatically exploreS` or `Our contextual banditS automatically explore` no?